app/javascript/src/components/Common/Item.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Avatar, Tooltip } from "neetoui";
+
+const Item = ({ note }) => {
+  return (
+    <div className="neeto-ui-shadow-s h-[146px] w-full p-4">
+      <div className="flex">
+        <Typography style="h4">{note.title}</Typography>
+        <div className="notes-card-dropdown">
+          <Dropdown
+            icon={MenuVertical}
+            label=""
+            position="right"
+            buttonStyle="text"
+          >
+            <li className="m-1">Edit</li>
+            <li>Delete</li>
+          </Dropdown>
+        </div>
+      </div>
+      <Typography style="body2" className="mb-3 text-gray-500">
+        {note?.description}
+      </Typography>
+      <hr />
+      <div className="mt-4 flex">
+        <button className="rounded-lg border-2  border-gray-900 bg-gray-50 py-1 px-2 text-xs text-gray-500">
+          Getting Started
+        </button>
+        <div className="ml-auto flex items-center space-x-2">
+          <Clock size={12} />
+          <Tooltip position="bottom" content={note.time}>
+            <Typography style="body3">Created {note.ago}</Typography>
+          </Tooltip>
+          <Avatar
+            onClick={function noRefCheck() {}}
+            size="small"
+            user={{
+              name: note?.createdBy,
```

Comment:
Is there a point in displaying the Avatar component itself if there is not `createdBy`? Currently wouldn't it show `undefined` in the Avatar if there is no `createdBy`? That we should avoid.

Reply:
Yes, If there is no` createdBy` entry in note object, then it will show undefined.
I thought it is better to use this instead of `note && note.createdBy`.

However in our code we're populating note as constant.

Like :
```
export const NOTES_DATA = [
  {
    id: 1,
    title: "How to Claim the warranty?",
    description:
      "Are you getting my texts??? she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting. Of course he was getting them, but if he wasn't getting",
    time: "Wdnesday 10.30AM",
    ago: "2 days ago",
    createdBy: "Oliver Smith",
  },
  { id: 2,
    title: "How to Claim the warranty?",
    description:
      "Are you getting my texts??? she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting. Of course he was getting them, but if he wasn't getting",
    time: "Wdnesday 10.30AM",
    ago: "2 days ago",
    createdBy: "Sachin Joshi",
  },
  ]
```

Reply:
>I thought it is better to use this instead of note && note.createdBy.

I am not talking about optional chaining here. Please re-read my review once again.

@sachinjoshi12 The question was why are you even rendering the `Avatar` component itself if there is no value for `createdBy`.